src/Compilers/Core/MSBuildTask/Shared/ManagedCompiler.cs

```diff
@@ -60,6 +60,9 @@ public ITaskItem[] Analyzers
         // We do not support BugReport because it always requires user interaction,
         // which will cause a hang.
 
+        /// <summary>
+        /// An instrument flag that specifies classes for which the compiler should generate instrumentation information.
```

Comment:
Please add help for this switch into the string that is printed by ```csc /?``` and ```vbc /?```. Also, clarify how multiple types can be specified. Should they be separated? With what? Etc.